src/plugins/imagery/components/ImageryView.vue

```diff
@@ -219,6 +223,8 @@ const ZOOM_SCALE_DEFAULT = 1;
 const SHOW_THUMBS_THRESHOLD_HEIGHT = 200;
 const SHOW_THUMBS_FULLSIZE_THRESHOLD_HEIGHT = 600;
 
+const LAYOUT_NAMES = new Set(['Display Layout', 'Flexible Layout']);
```

Comment:
A naive approach to detecting the composable view types. Open to more efficient alternatives, including extending the object api.

Reply:
Yeah, probably should add an `isComposableView` or some such to the ObjectAPI.

Reply:
You should explore this API: this.openmct.router.isNavigatedObject(objectPath)